docs/framework/20sources/30implementation/access-control.md

```diff
@@ -1,7 +1,56 @@
 ### Role Based Access Control
-Role Based Access Control (RBAC) allows separate accounts to be 
-responsible for different actions. It allows sensitive operations 
-to be controlled by one multisig while other less sensitive 
-operations can be controlled by a different multisig. 
+[Role Based Access Control (RBAC)](https://en.wikipedia.org/wiki/Role-based_access_control) 
+allows different entites to be responsible for different configuration actions. With 
+contracts, this can be used to limit which accounts can execute which functions. For 
+example, imagine a contract that operates as a crosschain bridge. It could have a role 
+called **PAUSER**. This role could be required to call a function that enables 
+pausing of the contract. Any transaction submitted by an account that did not 
+have the **PAUSER** role would be reverted.
 
-TODO: Provide example.
+Simplistic contracts might have a single role, **OWNER**, that can only be assigned
+to one account. For these contracts, the **owner** account is the only account 
+that can submit transactions that call configuration functions without reverting.
+
+The greater degree of flexibility afforded by Role Based Access Control compared 
+to simplistic **OWNER** style access control has security implications. For example, 
+a contract might be able to mint new tokens, and thus have a **MINTER** role to control
+this action. Minting new tokens could change the tokenomics of the contract, and hence 
+must only be executed if there is agreement between administrators. Access to this 
+configuration action might be limited to a multi-signature wallet account. The same 
+contract might have a **PAUSER** role that can be used to stop data processing within 
+the contract. The action to pause the contract needs to occur as quickly as possible,
+to halt an in-progress attack. However, access to the role needs to be limited to trusted
+accounts, to prevent attackers causing a Denial of Service attack on the contract, by
+continually pausing the contract. Using a multi-signature wallet to control this action
+is not ideal, as multiple parties would need to work together to pause the contract, thus 
+allowing attacks to continue longer than they otherwise would. In this situation, 
+multiple trusted accounts could be granted **PAUSER** role. Any one of the accounts 
+could then pause the contract.
+
+For a small project, when a contract is deployed, it might be tempting to use 
+simplistic **OWNER** style access control. 
+However, it is better to deploy a contract configured for fine grain
```

Comment:
I think it is worth qualifying here that this setup, which from a security perspective is equivalent to having a single `owner` based access control, does not offer any of the security benefits of a properly configured RBAC policy discussed in the first paragraph above.

Reply:
Agreed